Peekabook/Peekabook/Presentation/Recommend/VC/RecommendTableViewCell.swift

```diff
@@ -0,0 +1,193 @@
+        $0.layer.shadowRadius = 4
+        $0.layer.shadowOpacity = 0.3
+    }
+    private let bookRecommendedPersonImage = UIImageView().then {
+        $0.layer.cornerRadius = 50
+    }
+    private let bookRecommendedPersonLabel = UILabel().then {
+        $0.text = "인영케이"
+        $0.font = .systemFont(ofSize: 10, weight: .medium)
+        $0.textColor = .peekaRed
+    }
+    private let bookRecommendDateLabel = UILabel().then {
+        $0.text = "2022.12.25"
+        $0.font = .systemFont(ofSize: 9, weight: .medium)
+        $0.textColor = .peekaRed
+    }
+    
+    private let bookRecommendTextLabel = UILabel().then {
+        $0.numberOfLines = 8
+        $0.text = "추천문구샬라샬라"
+        $0.font = .systemFont(ofSize: 10, weight: .medium)
+        $0.textColor = .peekaRed
+    }
+    
+    private lazy var toFriendBookShelfButton = UIButton().then {
+        $0.backgroundColor = .peekaWhite
+        $0.layer.borderWidth = 1
+        $0.layer.borderColor = UIColor.peekaRed.cgColor
+        $0.setTitle("\(bookRecommendedPersonLabel.text ?? "누구세요")님의 책장 보러가기 →", for: .normal)
+        $0.setTitleColor(UIColor.peekaRed, for: .normal)
+        $0.titleLabel?.font = .systemFont(ofSize: 10, weight: .bold)
+        $0.contentHorizontalAlignment = .right
+        $0.contentEdgeInsets = UIEdgeInsets(top: 7, left: 10, bottom: 7, right: 12)
+    }
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        addSubview()
+        setLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        contentView.frame = contentView.frame.inset(by: UIEdgeInsets(top: 5, left: 20, bottom: 5, right: 20))
+    }
+}
+
+extension RecommendTableViewCell {
+    func addSubview() {
+        contentView.addSubviews([bookHeaderView, recommendStackView])
+        bookHeaderView.addSubviews([
+            bookNameLabel,
+            bookDividerLabel,
+            bookWriterLabel
+        ])
+        recommendStackView.addArrangedSubviews(bookImageContainerView, bookCommentsContainerView)
+        bookCommentsContainerView.addSubview(toFriendBookShelfButton)
+        bookImageContainerView.addSubview(bookImage)
+        bookCommentsContainerView.addSubviews([
+            bookRecommendedPersonImage,
+            bookRecommendedPersonLabel,
+            bookRecommendDateLabel,
+            bookRecommendTextLabel
+        ])
+    }
+
+    func setLayout() {
+        self.backgroundColor = .peekaBeige
+        
+        bookHeaderView.snp.makeConstraints { make in
+            make.top.leading.trailing.equalToSuperview()
+            make.height.equalTo(30)
+        }
+        bookNameLabel.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.leading.equalTo(15)
+        }
```

Comment:
p5.
요기부터 밑부분 한 줄씩 띄워주면 어떨까요?
```suggestion
        bookHeaderView.snp.makeConstraints { make in
             make.top.leading.trailing.equalToSuperview()
             make.height.equalTo(30)
         }
         
         bookNameLabel.snp.makeConstraints { make in
             make.centerY.equalToSuperview()
             make.leading.equalTo(15)
         }
```